Frontend/src/app/shared/components/navbar/navbar.component.ts

```diff
@@ -17,8 +17,8 @@ export class NavbarComponent implements OnInit {
   userInfo: PayloadDTO;
 
   constructor(
-    private router: Router,
-    private jwtService: JwtService,
+    public router: Router,
+    public jwtService: JwtService,
```

Comment:
Why switch these to public?

Reply:
Should probably not inline them on the html and just make functions

Reply:
I needed to access those in the HTML file and in order to do that, you have to make them public